contract-testing/spring-cloud-contract-messaging/provider/src/contractTest/resources/contracts/publish-clean-code.kts

```diff
@@ -0,0 +1,29 @@
+package contracts
+
+import org.springframework.cloud.contract.spec.ContractDsl.Companion.contract
+
+arrayOf(
+    contract {
+        name = "publish_clean_code"
+        label = "a_create_event_for_Clean_Code_is_published"
+
+        input {
+            triggeredBy = "publishCleanCode()"
+        }
+
+        outputMessage {
+            sentTo = sentTo("book-created")
+            body = body(
+                """
+                {
+                    "type": "book-created",
+                    "book": {
+                        "title": "Clean Code",
+                        "isbn": "9780132350884"
+                    }
+                }
+            """.trimIndent()
```

Comment:
Not very important, but I'd like line 25 indented one more level.

Reply:
done